lms/static/scripts/frontend_apps/utils/grader-service.js

```diff
@@ -16,11 +16,10 @@ import { apiCall } from './api';
 /**
  * Submits a student's grade to the LTI endpoint.
  *
- * @param {Student} student - Student object
- * @param {number} grade - A number between 0 and 1
- * @param {string} authToken - The auth token from the config
- * @return {Object} - An empty object
- *
+ * @param {Object} options
+ * @param {Student} options.student - Student object
```

Comment:
Should this be indented?

Reply:
Indeed it should. I think I wrote this before we adopted that convention.